glacier/src/exporters/svg.ts

```diff
@@ -0,0 +1,47 @@
+import * as vl from "vega-lite";
+declare global {
+    interface Element {}
+}
+import * as vega from "vega";
+import redux = require("redux");
+import {ModelState} from "../";
+import {Exporter} from "./";
+
+export function createSvgExporter(store: redux.Store<ModelState>) {
```

Comment:
In Redux, there is only one store. Functionality is isolated to specific scope parts by combining reducers on specific state keys. The values are to be stored within the store; Steve's been working on those actions separately.